modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -540,4 +540,5 @@ ifeval::["{context}" == "installing-ibm-power"]
 endif::[]
 ifeval::["{context}" == "installing-restricted-networks-ibm-power"]
 :!ibm-power:
+:!restricted:
```

Comment:
@alishaIBM I did a quick check of the Power docs and noticed this file doesn't show in your assemblies at all (both in the preview and the published docs) . Is that on purpose?

Reply:
@alishaIBM Any update here?

Reply:
@maxwelldb I went through the Power docs with @alishaIBM today and we found the content of that file in the preview. I only missed it because it didn't show in the leftnav toc. Reason is they used `leveloffset=+3` and not 2 like bare metal/IBM Z .  Everything else also looks good for the Power assemblies.

Reply:
@maxwelldb the IBM Power install assemblies look good. Thanks.

Reply:
Thanks! @SNiemann15 @alishaIBM I'll merge tomorrow unless I hear anything else.